doc/MigrationGuide_1_to_2.md

```
@@ -0,0 +1,72 @@
+
+CMCONF_SET(BA_PACKAGE_GIT_ARCHIVE_PATH_TEMPLATE "")
+CMCONF_SET(BA_PACKAGE_HTTP_HEADER "")
+CMCONF_SET(BA_PACKAGE_TEMPLATE_ARGS_URI_ESCAPE OFF)
+CMCONF_SET(BA_PACKAGE_URI_REVISION main)
+CMCONF_SET(BA_PACKAGE_URI_TEMPLATE_REMOTE "https://gitea.example.com/username/repository/media/<REVISION>/package/<GIT_PATH>/<PACKAGE_GROUP_NAME>/<ARCHIVE_NAME>")
+```
+
+Examples for different Package Repository hosting services and authentication methods can be found in [example/], [example-project] and [example-context]
+
+
+[CMCONF Global Config]: ./GlobalConfiguration.md
```

markdownlint-cli2 (0.21.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "cmconf global config"

(MD053, link-image-reference-definitions)